graphql/task_resolver.go

```diff
@@ -577,7 +577,7 @@ func (r *taskResolver) TaskLogs(ctx context.Context, obj *restModel.APITask) (*T
 func (r *taskResolver) Tests(ctx context.Context, obj *restModel.APITask, opts *TestFilterOptions) (*TaskTestResult, error) {
 	dbTask, err := task.FindOneIdAndExecution(utility.FromStringPtr(obj.Id), obj.Execution)
 	if err != nil {
-		return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error getting service model for APITask %s: %s", *obj.Id, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask %s: %s", *obj.Id, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
		return nil, InternalServerError.Send(ctx, fmt.Sprintf("getting service model for APITask '%s': %s", *obj.Id, err.Error()))
```